dom/src/MainDOMSource.ts

```diff
@@ -116,6 +117,7 @@ export class MainDOMSource implements DOMSource {
       this._namespace.concat(trimmedSelector);
     return new MainDOMSource(
       this._rootElement$,
+      this._sanitation$,
```

Comment:
I like the name sanitation, but I wonder if `cleanUp$` or `dispose$` might be more clear.